src/esp/gfx/MeshVisualizerDrawable.cpp

```diff
@@ -27,7 +27,7 @@ void MeshVisualizerDrawable::draw(const Magnum::Matrix4& transformationMatrix,
   shader_.setProjectionMatrix(camera.projectionMatrix())
       .setTransformationMatrix(transformationMatrix);
 
-  shader_.draw(mesh_);
+  shader_.draw(getMesh());
```

Comment:
Fail fast, same logic for every type of drawable. See my previous comments.

Reply:
Only the `GenericDrawable` codepath is possible when `createRenderer==true`. Therefore, `getMesh()` should definitely always succeed when it is called from here. So I don't think it's necessary to add some fail fast code here.

Reply:
See my previous comments. 

>Only the GenericDrawable codepath is possible when createRenderer==true.

A Typo? createRenderer!=true ?

Please read my previous comments first (about sanity check at the beginning of the draw function).

You need the same sanity check here simply because you modified the original code here even you understand this code path will never be reached. This is because other developers do not know this fact. If they do refactoring in the future, they can easily introduce bugs into the system.

Reply:
Oops yeah, that was a typo. Should be `createRenderer==false`.

This makes sense, I'll add in some earlier checks that `mesh_` exists. Thanks for the feedback!

Reply:
It's a bit inelegant because `mesh_` is private in order to prevent unsafe behavior since it's no longer a pointer. So the only possible way to sanity check would be to try running `getMesh()` and seeing if it errors.

Reply:
I saw the "CORRADE_ASSERT(glMeshExists(), ..." on the top of this function. It is great.